CodePush.js

```diff
@@ -409,6 +415,11 @@ if (NativeCodePush) {
             SYNC_IN_PROGRESS: 7, // There is an ongoing "sync" operation in progress.
             UNKNOWN_ERROR: -1
         },
+        UpdateState: {
+          RUNNING: NativeCodePush.codePushUpdateStateRunning,
+          PENDING: NativeCodePush.codePushUpdateStatePending,
+          LATEST: NativeCodePush.codePushUpdateStateLatest
+        },
```

Comment:
These too (here it's 2 spaces, but the CodePush object is defined with 4-spaced tabs)